app/views/questions/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Top Questions</h1>
+<ul class='list-group'>
+  <%= render @questions %>
```

Comment:
The common pattern here would be to each across `@questions` and to render the question (singular!) into a partial.